packages/suite/package.json

```diff
@@ -54,6 +54,7 @@
         "@trezor/analytics": "workspace:*",
         "@trezor/blockchain-link": "workspace:*",
         "@trezor/blockchain-link-types": "workspace:*",
+        "@trezor/blockchain-link-utils": "workspace:*",
```

Comment:
Is there a way around doing this? I can imagine trezor guys not being happy about this

Reply:
It depends on how you want to organise the code. It's either duplication or this. I don't know what trezor guys will prefer, so I just did it one way and want to wait and see what they say.